llvm/test/CodeGen/AMDGPU/GlobalISel/regbankselect.mir

```diff
@@ -186,23 +184,6 @@ body: |
     %1:_(s32) = G_LOAD %0 :: (load (s32) from %ir.tmp1)
 ...
 
----
-name: non_power_of_2
```

Comment:
Why was this deleted?

Reply:
Because this test assumes that the input is already legalized, but S448 is not legal.